src/covid_shared/cli_tools/run_directory.py

```diff
@@ -88,19 +88,20 @@ def setup_directory_structure(output_root: Union[str, Path], with_production: bo
 
 
 def make_links(app_metadata: Metadata, run_directory: Path,
-               mark_as_best: bool, production_tag: Optional[str]) -> None:
+               mark_as_best: bool, production_tag: Optional[str], quick: int) -> None:
```

Comment:
we need to think about all clients, not just ETL and snapshot.

Consider code searches for invocations to make_links and to finish_application:

https://stash.ihme.washington.edu/plugins/servlet/search?q=make_links
https://stash.ihme.washington.edu/plugins/servlet/search?q=finish_application

That's ~4 other repos we'd need to PR into.

Can we extract quick from app metadata? Not sure if it has input arguments stored. If so, that'd allow us to avoid interface change

Reply:
If you provide a default that recovers the original behavior for consumers, I think you're fine.  Both are API methods, so both need a default.

Reply:
I'm not sure I understand how we can recover that behavior. The consumer is not currently passing in 'quick' at all, and so a default of None will prevent a break but also won't work as intended, ie their quick runs will be marked 'latest'.

Reply:
Nevermind the above comment, I misunderstood what you meant by original behavior as original "intended" behavior.